FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Extensions/JObjectExtensions.cs

```diff
@@ -17,18 +19,26 @@ public static class JObjectExtensions
         /// </summary>
         /// <param name="resource">input resource.</param>
         /// <returns>The last updated timestamp of the resource.</returns>
-        public static DateTimeOffset? GetLastUpdated(this JObject resource)
+        public static DateTimeOffset GetLastUpdated(this JObject resource)
         {
-            string result =
-                (resource.GetValue(FhirBundleConstants.MetaKey) as JObject)?.Value<string>(FhirBundleConstants.LastUpdatedKey);
-            if (result == null)
+            try
             {
-                throw new FhirDataParseException("Failed to find lastUpdated value in resource.");
-            }
+                JObject lastUpdatedObject = JObject.FromObject(
```

Comment:
Seems directly call 
```csharp
            DateTimeOffset? result =
                (resource.GetValue("meta") as JObject)?.Value<DateTime>("lastUpdated");
```
will get similar results?

Reply:
Updated it, thanks!